manifests/j/JTL-SoftwareGmbH/JTL-Wawi/1.7.8.1/JTL-SoftwareGmbH.JTL-Wawi.installer.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: JTL-SoftwareGmbH.JTL-Wawi
+PackageVersion: 1.7.8.1

```

```suggestion
PackageVersion: 1.7.8.1
Scope: machine
```
Package installs in machine scope